test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -566,9 +566,112 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			By("Saving the old performance profile")
 			initialProfile = profile.DeepCopy()
 		})
+		When("Realtime enabled", func() {
+			It("[test_id:50990][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Should updated kernel arguments and tuned", func() {
+				By("Modifying profile")
+				profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
+					HighPowerConsumption: pointer.Bool(false),
+					RealTime:             pointer.Bool(true),
+				}
+				By("Updating the performance profile")
+				profiles.UpdateWithRetry(profile)
+
+				By("Verifying that mcp is ready for update")
+				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
```

Comment:
I'm a bit concerned that hardcoding the expected sequence so stricly can be fragile. Since this will always require a reboot (will it?) perhaps better wait for `Updating`, then for `Updated` again.